routes/merged_pr_page.py

```diff
@@ -8,8 +8,9 @@ def get_github_merged_pr(reload_data):
     """
     Get GitHub merged pull requests from a file or download new data.
     """
-    if config.GITHUB_MERGED_PR_LIST.is_file() and not reload_data:
-        with open(config.GITHUB_MERGED_PR_LIST, mode="r", encoding="utf-8") as file:
-            return json.load(file)
+    if config.GITHUB_TOKEN:
+        if not config.GITHUB_MERGED_PR_LIST.is_file() or reload_data:
+            return github_api.get_merged_pull_request()
 
-    return github_api.get_merged_pull_request()
+    with open(config.GITHUB_MERGED_PR_LIST, mode="r", encoding="utf-8") as file:
```

Comment:
This is gonna fail if I don't have the token and the file does not exist. Other pages handle this by returning an empty dict.

Reply:
good catch, thank you